lookout/core/manager.py

```diff
@@ -7,6 +7,7 @@
 from lookout.core.data_requests import DataService
 from lookout.core.event_listener import EventHandlers
 from lookout.core.model_repository import ModelRepository
+from lookout.style.typos.correction_model import ContextXgbModel
```

Comment:
Yes, I agree. But this was done on purpose because now the pretrained model loading takes too long, and it's better to do it once on running the analyzer. If do it in analyzer.__init__(), model will be loaded on every review request. There might be a better way to do it, I'll think about it, this was a quick hard coding for quicker testing.

Reply:
It might be better as a class attribute then!